testdata/fork/Fork.t.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity >= 0.8.0;
+
+import "ds-test/test.sol";
+
+interface Cheat {
+    function store(address account, bytes32 slot, bytes32 value) external;
+}
+
+interface erc20 {
+    function balanceOf(address) external returns (uint);
+    function totalSupply() external returns (uint);
+    function transfer(uint,address) external;
+}
+
+contract TestContract {
```

Comment:
Can we make the name of this contract more verbose?

Reply:
This test was a bit confusing to me. You said you want a test that deploys a contract, does deploying contracts behave differently when running against the forked state? Or is this just to make sure that forking didn't introduce some new weird behaviour?

Reply:
The tests in `testdata` are tests that I wrote when porting to REVM where I basically rebuilt the test suite from scratch to make sure that every behavior I expected to hold true holds true (as best as I could) - currently, there is no odd behavior when we deploy contracts in fork mode, but conceivably if we made some bigger changes to REVM or ported to something else then this might not be the case

If this is for the deployment test I think something like 'DummyContract' or something is better, but it's a matter of opinion I guess :)